.changeset/nice-kangaroos-kiss.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rhds/elements": minor
+---
+
+`<rh-pagination>`: Added dark mode support
```

Comment:
```suggestion
`<rh-pagination>`: Added dark color palette
```
A ways back, we decided on the terminology "color palette" and "theme" to represent color stuff in our system. The docs on this are not quite up to date yet